### PR TITLE
Working on User Groups functionality

### PR DESCRIPTION
**Pull request by aalex121**
Working on User Groups functionality

### CODE REVIEW

File: src/app/DataModels/userMessageRequestModel.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+export interface UserMessageRequestModel {
```
Comment (DoctorChaOs96) [inline src/app/DataModels/userMessageRequestModel.ts:1]: Usual naming convention is ISomething.

File: src/app/create-user-group/create-user-group.component.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<form class="form-signin">
+    <h4>
+        Create User Group
+    </h4>    
+
```
Comment (DoctorChaOs96) [inline src/app/create-user-group/create-user-group.component.html:5]: 5th and 10th empty rows are redundant.

File: src/app/messaging/messaging.component.ts [hunk 3]
```diff
@@ -17,41 +17,37 @@ export class MessagingComponent implements OnInit {
...
-        );
-  }
-
-  getMessagesFromTo(): void {
+  displayMessages: MessageResponseModel[];
+  displayMessagesOffset: number = 0;  
+
+  //TODO Fill with actual values!
+  getMessagesRequest: UserMessageRequestModel = {
+    messageType: 0,
```
Comment (DoctorChaOs96) [inline src/app/messaging/messaging.component.ts]: I believe you have enum for messageTypes.

File: src/app/DataModels/userMessageRequestModel.ts (hunk 1 above)
Comment (aalex121) [reply src/app/DataModels/userMessageRequestModel.ts:1]: From the code examples at https://angular.io/docs, it looks like the Hungarian notation is not used in Angular JS apps.

File: src/app/create-user-group/create-user-group.component.html (hunk 2 above)
Comment (aalex121) [reply src/app/create-user-group/create-user-group.component.html:5]: Corrected.

File: src/app/messaging/messaging.component.ts (hunk 3 above)
Comment (aalex121) [reply src/app/messaging/messaging.component.ts]: Corrected.

File: src/app/DataModels/newGroupModel.ts [hunk 4]
```diff
@@ -1,4 +1,4 @@
 export interface NewGroupModel {
     creatorId: number;
-    groupName: string;
+    groupName: string;
```
Comment (DoctorChaOs96) [inline src/app/DataModels/newGroupModel.ts:3]: Redundant change

**Comment by DoctorChaOs96** [comment]
I'm OK. You can start to merge.

**Pull request closed without merging**